src/main/resources/META-INF/aop.xml

```diff
@@ -3,10 +3,17 @@
 <aspectj>
   <aspects>
     <aspect name="kamon.jdbc.instrumentation.StatementInstrumentation"/>
+    <aspect name="kamon.jdbc.instrumentation.HikariInstrumentation"/>
   </aspects>
 
-    <weaver>
-       <include within="java.sql.Statement..*"/>
-       <include within="java.sql.Connection..*"/>
-   </weaver>
+
+  <weaver>
+    <include within="com.zaxxer.hikari.pool.HikariPool"/>
+    <include within="com.zaxxer.hikari.pool.ProxyConnection"/>
+
+    <!-- JDBC Drivers -->
+    <include within="org.h2..*"/>
+    <include within="org.postgresql.jdbc4..*"/>
+    <include within="com.microsoft.sqlserver.jdbc..*"/>
```

Comment:
of course! I just didn't have the packages at hand.. also, I'm not sure if there is a way in which we could automate testing those :/